src/study_plans/services.py

```
@@ -71,96 +71,55 @@ def create_study_plan(self, plan_data: dict, workspace_info: Dict = None) -> str
+                "institute_id": None,
+                "subject_id": None,
+                "status": "active",
+                "created_at": datetime.utcnow(),
+                "updated_at": datetime.utcnow()
+            }
+            study_plan = StudyPlanPerSubject(**plan_data)
+            result = self.collection.insert_one(study_plan.to_dict())
+            return str(result.inserted_id)
         except Exception as e:
-            raise AppException(f"Error al actualizar plan: {str(e)}", AppException.BAD_REQUEST)
+            raise AppException(f"Error al crear plan personal: {str(e)}", AppException.BAD_REQUEST)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)